virttest/utils_disk.py

```diff
@@ -600,6 +602,154 @@ def create_filesystem(session, partition_name, fstype, ostype, timeout=360):
         create_filesyetem_linux(session, partition_name, fstype, timeout)
 
 
+def get_fstype_mpoint_linux(session, partition):
+    """
+    Get a partition's file system type and mountpoint.
+
+    :param session: session object to guest.
+    :param partition: disk partition, like /dev/sdb1.
+    :return: Tuple (fstype, mountpoint)
+    """
+    mount_list = session.cmd_output('cat /proc/mounts')
+    pattern = r'%s\s(.+?)\s(.+?)\s' % partition
+    mount_info = re.search(pattern, mount_list)
+    if mount_info:
+        mountpoint = mount_info.group(1)
+        fstype = mount_info.group(2)
+        return (fstype, mountpoint)
+    logging.error('partition is not mounted, mounted'
+                  'list as:\n%s' % mount_list)
+
+
+def resize_filesystem_linux(session, partition, size):
+    """
+    resize file system in linux guest.
+    support enlarge and shrink.
+
+    :param session: session object to guest.
+    :param partition: disk partition, like /dev/sdb1.
+    :param size: resize file system to size, unit is B.
+                 support transfer size with AVAILABLE_SIZE,
+                 enlarge to maximun available size.
+    """
+    fstype, mountpoint = get_fstype_mpoint_linux(session, partition)
+    if fstype == 'xfs':
```

Comment:
Suggest to add comments to doc string.  xfs doesn't support shrink.

Reply:
that's ok, will updated~